tsconfig.json

```diff
@@ -19,6 +19,7 @@
     "resolveJsonModule": true,
     "outDir": "lib",
     "rootDir": "src",
+    "skipLibCheck": true,
```

Comment:
Need to investigate the following TypeScript build issue:

```
node_modules/zod/v3/index.d.ts:1:15 - error TS2846: A declaration file cannot be imported without 'import type'. Did you mean to import an implementation file '../dist/types/v3/index.js' instead?

1 export * from "../dist/types/v3/index.d.ts";
                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/zod/v4/core/index.d.ts:1:15 - error TS2846: A declaration file cannot be imported without 'import type'. Did you mean to import an implementation file '../../dist/types/v4/core/index.js' instead?

1 export * from "../../dist/types/v4/core/index.d.ts";
                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
This seems to be fixed https://github.com/colinhacks/zod/issues/4545
Working fine with `resolutions` pining it to `"^3.25.56"`.

Reply:
Oh cool, thanks!

Reply:
Looks like the latest release may have these changes now: https://github.com/langchain-ai/langchainjs/releases/tag/langchain%3D%3D0.3.29